src/client/automation/playback/scroll.js

```diff
@@ -266,9 +266,7 @@ export default class ScrollAutomation {
     }
 
     static _getFixedAncestorOrSelf (element) {
-        const ancestors = domUtils.getAncestorsAndSelf(element).reverse();
-
-        return arrayUtils.find(ancestors, el => styleUtils.isFixedElement(el));
+        return domUtils.findParent(element, true, node => styleUtils.isFixedElement(node));
```

Comment:
can it be just ` domUtils.findParent(element, true, styleUtils.isFixedElement);`